scipy/linalg/_decomp.py

```diff
@@ -336,18 +338,26 @@ def eigh(a, b=None, lower=True, eigvals_only=False, overwrite_a=False,
         Whether to check that the input matrices contain only finite numbers.
         Disabling may give a performance gain, but may result in problems
         (crashes, non-termination) if the inputs do contain infinities or NaNs.
-    turbo : bool, optional
-        *Deprecated since v1.5.0, use ``driver=gvd`` keyword instead*.
+    turbo : bool, optional, deprecated
         Use divide and conquer algorithm (faster but expensive in memory, only
         for generalized eigenvalue problem and if full set of eigenvalues are
-        requested.). Has no significant effect if eigenvectors are not
+        requested). Has no significant effect if eigenvectors are not
         requested.
-    eigvals : tuple (lo, hi), optional
-        *Deprecated since v1.5.0, use ``subset_by_index`` keyword instead*.
+
+            .. deprecated:: 1.5.0
+                `eigh` keyword argument `turbo` is deprecated in favour of
+                ``driver=gvd`` keyword instead and will be removed in SciPy
+                1.12.0.
+    eigvals : tuple (lo, hi), optional, deprecated
         Indexes of the smallest and largest (in ascending order) eigenvalues
```

Comment:
Same with this text. No need to mention why it was here as we want people drawn to the `subset_by_index`.